core/ts/Poll.ts

```diff
@@ -751,21 +767,23 @@ export class Poll implements IPoll {
       this.sbSalts[this.currentMessageBatchIndex!],
     ]);
 
+    const inputBatchHash = this.batchHashes[index - 1];
+    const outputBatchHash = this.batchHashes[index];
+
     // Generate a SHA256 hash of inputs which the contract provides
     /* eslint-disable no-bitwise */
     const packedVals =
       BigInt(this.maxValues.maxVoteOptions) +
       (BigInt(this.numSignups) << 50n) +
-      (BigInt(index) << 100n) +
+      (BigInt(0) << 100n) +
```

Comment:
why is this always 0? is it something we need then?

Reply:
0 represent index of first message in current batch (batchStartIndex) which is always zero. We can change that if you want but we will need to change functions that pack values with bits operations.

Reply:
so here https://github.com/privacy-scaling-explorations/maci/blob/fb2a3270e44b79edc4cd9799cdb72b3a5eee9636/contracts/contracts/MessageProcessor.sol#L260 in the contract we always do (0 * messageBatchsize) << 100?

Reply:
this is gonna be addressed in another PR